.gitignore

```diff
@@ -79,3 +79,7 @@ bin/*
 
 # External downloaded libs (TPL downloaded libs, etc...)
 external_libraries/tetgen/*
+
+# Inclusion of post files for testing
+!**/auxiliar_files_for_python_unittest/**/*.post.res
```

Comment:
This should not be here, the files should be removed by the test

Reply:
> This should not be here, the files should be removed by the test

Then, this [file](https://github.com/KratosMultiphysics/Kratos/blob/master/kratos/tests/auxiliar_files_for_python_unittest/gid_io/ref_gid_io_gp_active_only_0.post.msh) and some others must be deleted

Let me check what's happening with the test